README.md

```diff
@@ -103,43 +100,39 @@ development teams - a skill which is currently in a very high demand.
             (`maven-archetype-quickstart`).
 
         2.  **[Spring](http://spring.io/)** is probably the best Java framework
-        to begin with. It is used to quickly build web applications. And it is
-        very popular.
+        to begin with. It is a popular way to quickly build web applications.
 
-        There's a book which helps start with Spring -
+        This book is useful for getting started with Spring -
         "[Spring in action](https://www.manning.com/books/spring-in-action-fourth-edition)".
-        But it's probably also a good opportunity for us to stress that
-        another thing that makes you a better security professional is **reading
-        reference manuals**, so we do strongly recommend you to read spring
-        reference documents in addition to the book.
-
-        This is probably the **hardest item to learn on this list**, so please
-        allow some time for reading and practice. But the knowledge and
-        practice obtained while learning Spring is exactly what will make you
+        This is a good opportunity for us to stress that
+        **reading reference manuals** will help make you a better security professional,
+        so we recommend that you to read Spring reference documentation in addition to the book.
+
+        We consider this to be the **hardest item to learn on this list**, so
+        allow some time for reading and practice. The knowledge and
+        practice obtained while learning Spring will make you
         stand out as an application security professional.
 
 
     2. Scripting language (secondary programming language): **ECMAScript aka
        JavaScript** (specifically, **[NodeJS](https://nodejs.org/)**)
 
-       In addition to the main programming langunage, a security professional
-       will always need to write scripts: It can be a quick test, or some
-       automation, or maybe even a solution that injects security into a build
-       pipeline. It will involve items commonly at a level of operating system
+       In addition to their main programming langunage, a security professional
+       will need to write scripts: this can be a quick test, some
```

Comment:
Just FYI: picked up the 'langunage' typo in a separate future PR. Waiting for you you to accept the invite into this repo so you can review and approce =)